morpheus-graphql-core/src/Data/Morpheus/Rendering/RenderGQL.hs

```diff
@@ -59,6 +63,9 @@ instance RenderGQL Bool where
   render True = "true"
   render False = "false"
 
+instance RenderGQL A.Value where
+  render x = decodeUtf8 $ toStrict $ A.encode x
```

Comment:
@nalchevanidze I wanted to check with you about this. This is potentially inefficient because Aeson has to encode to `ByteString`, and then we need to decode it to `Text`. Is there a reason `RenderGQL` needs to render to `Text`? Most things I've seen that strive to be high-performance (Aeson, WAI, etc.) just use `ByteString`.

Reply:
sometimes i also considering to mograte types to `Bytestring` or even make them type agnostic. 

like:

```hs
class RenderGQL a out where
  render :: a -> out
```
you can open a new issue to discuss this topic.